GateWatcher/aioniq/_meta/fields.yml

```diff
@@ -0,0 +1,84 @@
+gatewatcher.type:
```

Comment:
I wonder if this field could be an [observer.type](https://www.elastic.co/guide/en/ecs/current/ecs-observer.html#field-observer-type)

Reply:
The semantic of `observer.type` seems different from `gatewatcher.type`